src/WormholeRouter.sol

```diff
@@ -121,9 +128,9 @@ contract WormholeRouter {
      * @param maxFeePercentage Max percentage of the withdrawn amount (in WAD) to be paid as fee (e.g 1% = 0.01 * WAD)
      */
     function requestMint(WormholeGUID calldata wormholeGUID, uint256 maxFeePercentage) external {
-        require(msg.sender == gateways[wormholeGUID.sourceDomain], "WormholeRouter/sender-not-gateway");
+        if (msg.sender != gateways[wormholeGUID.sourceDomain]) revert SenderNotGateway(msg.sender, gateways[wormholeGUID.sourceDomain]);
         address gateway = gateways[wormholeGUID.targetDomain];
-        require(gateway != address(0), "WormholeRouter/unsupported-target-domain");
+        if (gateway == address(0)) revert UnsupportedTargetDomain(gateway);
```

Comment:
This should be `revert UnsupportedTargetDomain(wormholeGUID.targetDomain);` as `gateway` will always be 0 in this case.

Reply:
done in 0172295